contracts/eden/src/actions/accounts.cpp

```diff
@@ -91,6 +95,50 @@ namespace eden
       eosio::check(migrations{get_self()}.is_completed<migrate_account_v0>(),
                    "Tables must be migrated to enable donations");
       accounts{get_self()}.sub_balance(owner, quantity);
-      accounts{get_self(), "owned"_n}.add_balance("master"_n, quantity);
+      add_to_pool(get_self(), "master"_n, quantity);
    }
+
+   void eden::distribute(uint32_t max_steps)
+   {
+      eosio::check(distribute_monthly(get_self(), max_steps) != max_steps, "Nothing to do");
+   }
+
+   void eden::fundtransfer(eosio::name from,
+                           eosio::block_timestamp distribution_time,
+                           uint8_t rank,
+                           eosio::name to,
+                           eosio::asset amount,
+                           const std::string& memo)
+   {
+      eosio::check(amount.amount > 0, "amount must be positive");
+      eosio::check(memo.size() <= 256, "Memo has more than 256 bytes");
+      eosio::require_auth(from);
+      members members{get_self()};
+      members.check_active_member(from);
+      members.check_active_member(to);
+      distribution_account_table_type distribution_accounts{get_self(), default_scope};
+      accounts accounts{get_self()};
+      const auto& row = distribution_accounts.get_index<"byowner"_n>().get(
+          distribution_account_key(from, distribution_time, rank));
+      eosio::check(row.balance() >= amount, "insufficient balance");
+      distribution_accounts.modify(row, get_self(), [&](auto& row) { row.balance() -= amount; });
```

Comment:
The row sticks around forever with an empty balance, unless `on_election_kick` or `on_resign` happens. Is this intended?